### ISSUE TITLE
Login Design

### ISSUE THREAD

**Issue by hifabienne** [zitadel/zitadel#5486]

**Comment by juergrinaldi** [comment]
As discussed with @livio-a, the most efficient approach might be to implement the design for Step 1 as 'status quo' (when nobody has any objections) to save some time. If things to change occur we can react on the fly.

In the meantime I'll have a look at Step 2 and 3 while 1 is in developement and see if any changes are needed.

IDP and MFA are the ones that catch my eyes the most, but...
1) for IDP there's a pretty recent design concept done by me (obviously not based on metrics/feedback), so I'd suggest we run with that and...
2) for MFA we had at least some feedback from the user test from a while ago, where we already had some improvements put in. But this might still be the most confusing one for new users to this concepts.

TL;DR: Start developement for step 1 immediately and I can give inputs on the fly, while I work on Step 2 and 3 simultaneously.

**Comment by hifabienne** [comment]
@juergrinaldi Sounds good for me. 
Just one note. Yes step 1 is the first that will be implemented, but that sprint starts on the 12. April. If we can stick to the plan. This means that if you like to do some design for it you would have time. It should be finished by the 7. April, so we can take it into the refinement.

**Comment by hifabienne** [comment]
@juergrinaldi Do you already have something?
